crossdock/server/server.go

```diff
@@ -77,6 +85,9 @@ func (s *Server) Start() error {
 			return s.doJoinTrace(ctx, req.(*tracetest.JoinTraceRequest))
 		})
 	})
+	// TODO test_driver host needs to be initialized before this
+	//mux.HandleFunc("/endtoend", s.eHandler.Trace)
```

Comment:
must it? docker-compose can just as easily run another container for the agent, which will give you a stable host name.

otherwise, I assume you can just pass an env variable to this server with the host name for the agent.